Align docs with MC 5.4.0

emrberk opened the pull request:
- Screenshots are updated with rebranding.
- Top Bar and Side Bar updates are added.
- License pages are updated.
- Some tables are updated.
- Removed legacy features.
- Removed screenshots about legacy features.

rebekah-lawrence reviewed (commented):
Before I can do a proper review, I need to understand why the change in name from toolbar and menu to Top Bar and Side Bar. 

I do not see this change as providing any further information to users - potentially Title Bar and Side Navigation are clearer, or even Top Menu and Side Menu/Navigation. In fact, we don't really have to name the toolbar (and I agree, it isn't a toolbar), we can just say Select x, located at the top of the screen, or Select x from the options at the top of the screen.

In general, however, these must be referred to as "the x Bar", not just "x Bar"

This will be a job for the IA/style guide/doc improvements. Noting for @docs team; nothing for you to do. Click ought to be changed to Select throughout. However, I see that Click is used across the MC docs, so it is not worth changing in just these updates. In the same area, per is Latin, ought to be for each.

emrberk commented:
> Before I can do a proper review, I need to understand why the change in name from toolbar and menu to Top Bar and Side Bar.
> 
> I do not see this change as providing any further information to users - potentially Title Bar and Side Navigation are clearer, or even Top Menu and Side Menu/Navigation. In fact, we don't really have to name the toolbar (and I agree, it isn't a toolbar), we can just say Select x, located at the top of the screen, or Select x from the options at the top of the screen.
> 
> In general, however, these must be referred to as "the x Bar", not just "x Bar"
> 
> This will be a job for the IA/style guide/doc improvements. Noting for @docs team; nothing for you to do. Click ought to be changed to Select throughout. However, I see that Click is used across the MC docs, so it is not worth changing in just these updates. In the same area, per is Latin, ought to be for each.

Hi Rebekah,

Top Bar (previously referred as Toolbar) does not contain any tools but selecting cluster right now. That's why I changed the naming here. Side Bar contains SQL browser and Console buttons as well as Menu. So, We can't call it "menu" now, because Menu is just a part of Side Bar and can be opened/closed. Side Bar and Top Bar may not be the best naming, and one can come up with better names for them. I just referred them as in our repository, so feel free to revise it.
Luckily, we don't have so many text updates in this PR, but mostly screenshots :)

rebekah-lawrence commented:
I've requested @pawelklasa's review - he will doubtless have better ideas than I on how we should refer to the UI elements.

jgardiner68 commented:
@emrberk Screen shots look good.   The bulk are updated images but there are more deletions than additions.  Have we confirmed that this is not removing useful images from the docs?

jgardiner68 commented on docs/modules/data-structures/pages/map.adoc:
Backups, Locks and Dirty Entries are still available but have been added to another view to simplify the layout.

emrberk replied:
Added them again, please take a look.

jgardiner68 commented on docs/modules/clusters/pages/clients.adoc:
```diff
@@ -20,9 +20,6 @@ image:ROOT:ClientGridView.png[Client Grid View]
 |Type
 |Type of client. This field usually contains the client's programming language.
 
-|Enterprise
```
Why has this been removed?  IIRC we still have EE and non-EE Java clients (The change was to merge the Enterprise column into another column)

emrberk replied:
They are merged to "Type" column, shown as "Java Enterprise". So this column does not exist anymore.

emrberk replied:
Added the description about Enterprise to Type column, can you please take a look at it?

emrberk replied:
>What happened to client labels?

It exists in Client Labels View. Explanation is still present.

jgardiner68 commented on docs/modules/clusters/pages/cluster-license.adoc:
```diff
@@ -5,14 +5,13 @@ You can use the *Cluster License* tab of the *Administration* menu item to see d
 - *License Key Hash*: SHA-256 hash of the key as a Base64 encoded string. The actual key is not displayed for security reasons.
 - *Expires*: The date on which the license will expire.
 - *Licensed Cluster Members*: The total number of members that the license supports and the current number of members running on the license.
-- *License Type*: The type of license.
```
has this info been moved elsewhere?

emrberk replied:
It doesn't exist anymore, at least not in this page.

jgardiner68 commented on docs/modules/cp-subsystem/pages/atomic-ref.adoc:
```diff
@@ -5,6 +5,4 @@ by clicking on the **Atomic References** menu item in the left menu.
 
 image:ROOT:AtomicReferences.png[Atomic References View]
 
-An atomic reference has only a single metric: Its name.
```
This is to avoid tickets about there being less detail -- I think it should be kept.

emrberk replied:
All CP data structures have name and group columns, but we don't refer to them as metrics. I think it makes sense not to refer name as metric.

jgardiner68 commented on docs/modules/data-structures/pages/map.adoc:
```diff
@@ -153,18 +153,15 @@ From left to right, this table lists the following:
 |Entry Memory
 |Memory cost of owned entries in the member.
 
-|Backups
```
Moved to a different view

emrberk replied:
Added.

jgardiner68 commented on docs/modules/data-structures/pages/map.adoc:
```diff
@@ -174,15 +171,6 @@ the member, including those which are no longer in the map (for
 example, may have been evicted). The number of hits may be inaccurate
 after a partition is migrated to a new owner member.
 
-|Locks
```
moved to a different view

emrberk replied:
Added.

jgardiner68 commented on docs/modules/data-structures/pages/map.adoc:
```diff
@@ -221,14 +212,17 @@ From left to right, this table lists the following:
 |Max Avg Get Latency
 |Maximum average latency of get operations on the member.
 
-|Max Avg Remove Latency
```
moved to a different view?

emrberk replied:
Not even in a different view, added it again.

jgardiner68 commented on docs/modules/deploy-manage/pages/updating-mc-license-ui.adoc:
```diff
@@ -15,18 +15,23 @@ This feature is available only to admin users.
 - *Licensed Cluster Members*: The total number of members that the license supports and the current number of members running on the license.
-- *License Type*: The type of license.
-+
-Warnings are displayed when a license expires within 30 days.
```
This is still true - where is it captured?

emrberk replied:
Added it again, in the same section.

jgardiner68 reviewed (changes requested):
Concerned that some sections have been removed from the docs whereas those columns have been moved to different views in the same table.  This should be explained.

Is there a doc page to explain the Licensed Features page?

emrberk commented:
> Concerned that some sections have been removed from the docs whereas those columns have been moved to different views in the same table. This should be explained.
> 
> Is there a doc page to explain the Licensed Features page?

Addressed the review comments about the removed columns. We don't have a doc page for promo.

rebekah-lawrence reviewed (approved):
Language is consistent with other MC content. Approving.

emrberk commented on docs/modules/data-structures/pages/map.adoc:
```diff
@@ -153,9 +153,12 @@ From left to right, this table lists the following:
 |Number of put operations received by the member.
 
-|Removes
+|Removals
```
The table column is named as "Removals". In other data structure pages, we also refer to it as "Removals", despite having "Removes/s" in throughput statistics. That's why I put "Removals" here.

emrberk replied:
>Wht about Backups, Backup Memory, Locks, Dirty Caches

Can you elaborate?
